base/client.jl

```diff
@@ -71,6 +71,18 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            # build correctly quoted string for cmd.exe
+            special = r".*[ &()[\]{}\^=;!'+,`~]+.*"
```

Comment:
Do you have a reference for this regex? IIRC, I've usually seen it suggested to use a single linear pass. FWIW, it appears that " may be missing from the list? The .* is unnecessary unless you are also attaching the regex to the ends with ^ and $.